### ISSUE TITLE
The Info.plist file is missing the required keys: CFBundleShortVersionString & CFBundleVersion

### ISSUE THREAD

**Issue by sergioprot** [AgoraIO/Agora-Flutter-SDK#609]
These errors are shown, when trying to upload app to Test Flight:

```
16:50:06 error: exportArchive: App Store Connect Operation Error. ERROR ITMS-90057: "The bundle 'Payload/Runner.app/Frameworks/AgoraRtcWrapper.framework' is missing plist key. The Info.plist file is missing the required key: CFBundleShortVersionString. Please find more information about CFBundleShortVersionString at https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleshortversionstring".
16:50:06 
16:50:06 Error Domain=ITunesTransporterErrorDomain Code=-18000 "error: App Store Connect Operation Error. ERROR ITMS-90057: "The bundle 'Payload/Runner.app/Frameworks/AgoraRtcWrapper.framework' is missing plist key. The Info.plist file is missing the required key: CFBundleShortVersionString. Please find more information about CFBundleShortVersionString at https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleshortversionstring"." UserInfo={NSLocalizedDescription=error: App Store Connect Operation Error. ERROR ITMS-90057: "The bundle 'Payload/Runner.app/Frameworks/AgoraRtcWrapper.framework' is missing plist key. The Info.plist file is missing the required key: CFBundleShortVersionString. Please find more information about CFBundleShortVersionString at [https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleshortversionstring".}](https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleshortversionstring%22.%7D)
16:50:06 
16:50:06 error: exportArchive: App Store Connect Operation Error. ERROR ITMS-90056: "This bundle Payload/Runner.app/Frameworks/AgoraRtcWrapper.framework is invalid. The Info.plist file is missing the required key: CFBundleVersion. Please find more information about CFBundleVersion at https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleversion".
16:50:06 
16:50:06 Error Domain=ITunesTransporterErrorDomain Code=-18000 "error: App Store Connect Operation Error. ERROR ITMS-90056: "This bundle Payload/Runner.app/Frameworks/AgoraRtcWrapper.framework is invalid. The Info.plist file is missing the required key: CFBundleVersion. Please find more information about CFBundleVersion at https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleversion"." UserInfo={NSLocalizedDescription=error: App Store Connect Operation Error. ERROR ITMS-90056: "This bundle Payload/Runner.app/Frameworks/AgoraRtcWrapper.framework is invalid. The Info.plist file is missing the required key: CFBundleVersion. Please find more information about CFBundleVersion at [https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleversion".}](https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleversion%22.%7D)
```

App uses agora_rtc_engine version 5.0.0.

**Comment by LichKing-2234** [comment]
[AgoraRtcWrapper.framework.zip](https://github.com/AgoraIO/Agora-Flutter-SDK/files/8158597/AgoraRtcWrapper.framework.zip)
You can try to replace this Framework on your disk before our publish the fixed version.

**Comment by sergioprot** [comment]
Thanks! How soon will the fixed version be published?

**Comment by dmitry-fbm** [comment]
> You can try to replace this Framework on your disk before we publish the fixed version.

It would be pretty difficult to do as the build is a part of our CI/CD flow, we'd need to redo it to replace automated update of dart packages with "hardcoded" version.

**Comment by demagomes** [comment]
Hi @LichKing-2234 where do these files go?

**Comment by demagomes** [comment]
Hi there, was this issue fixed in 5.1.0?

**Comment by dmitry-fbm** [comment]
> Hi there, was this issue fixed in 5.1.0?

Yes thanks, the ticket can be closed.
